models/ngo.model.js

```diff
@@ -2,15 +2,18 @@ const mongooose = require('mongoose');
 
 const NGOSchema = new mongooose.Schema({
   name: { type: String, required: true },
-  username: { type: String, required: true },
   password: { type: String, required: true },
   city: { type: String, required: true },
-  email: { type: String },
+  email: { type: String, required: true },
   phone_number: { type: String, required: true },
+  registration_number: { type: String, required: true },
   ngo_images: { type: Array },
   address: { type: String },
-  available: { type: Boolean, required: true, default: true },
-  available_items: { type: Map, of: Number, default: 0 },
+  is_available: { type: Boolean, required: true, default: true },
+  available_items: [{
+    category: { type: String, required: true },
+    item_count: { type: Number, default: 0 }
+  }],
```

Comment:
Instead of storing as available items as category and item count just storing the sectors field in which the NGO is working would be better.

Reply:
but we need to keep count of the items as well. Since, we are keeping stats of the availability of items at various NGOs.